.github/workflows/stress_test.yml

```diff
@@ -0,0 +1,36 @@
+name: Stress-test
+
+on: [pull_request]
```

Comment:
Is the `concurrency` configurations we have in our `iFixit/ifixit` `E2E.yml` only needed if we are using a `self-hosted` runner?

Reply:
> Is the concurrency configuration we have in our iFixit/ifixit E2E.yml only needed if we are using a self-hosted runner?

I believe we can add in react-commerce as well but since they're free I think we don't care too much. The reason why we have it on `iFixit/ifixit` is because we have a limited amount of runners.